examples/kafka_spark_demo/feature_repo/streaming_features_tutorial.ipynb

```diff
@@ -0,0 +1,877 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Building streaming features with Feast, Spark, and Kafka"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Overview\n",
+    "\n",
+    "This notebook explores how data scientists and engineers can build streaming features in Feast. It builds off of the first [module](https://github.com/feast-dev/feast-workshop/tree/main/module_1) from the Feast workshop. The workshop module is not a prerequisite, but is recommended if you are not already familiar with the Push API in Feast.\n",
+    "\n",
+    "A typical pattern that we see is that data scientists use Feast to define, use, and share features, which greatly improves their productivity, while ML engineers use Feast to ensure that all the features are available for both training and serving. This notebook will first show how a data scientist can define and use stream features in Feast, and then how an ML engineer can use the Push API to ensure that those streaming features are available in production.\n",
+    "\n",
+    "All the necessary resources to run this notebook are in `kafka_spark_demo`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Setup the feature store"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Apply feature repository\n",
+    "We first run `feast apply` to register the data sources and features."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/Users/felixwang/feast/env/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/specifiers.py:273: DeprecationWarning: Creating a LegacyVersion has been deprecated and will be removed in the next major release\n",
+      "  warnings.warn(\n",
+      "/Users/felixwang/feast/env/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/version.py:111: DeprecationWarning: Creating a LegacyVersion has been deprecated and will be removed in the next major release\n",
+      "  warnings.warn(\n",
+      "/Users/felixwang/feast/sdk/python/feast/stream_feature_view.py:98: RuntimeWarning: Stream Feature Views are experimental features in alpha development. Some functionality may still be unstable so functionality can change in the future.\n",
```

Comment:
I feel like it's strange that users will get warnings warning them that the tutorial they are going through it unstable 🤔

Reply:
yup, fixed